lib/ansible/modules/storage/glusterfs/gluster_peer.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# Copyright 2015 Nandaja Varma <nvarma@redhat.com>
+# Copyright 2018 Red Hat, Inc.
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: gluster_peer
+short_description: Attach/Detach peers to/from the cluster
+description:
+  - Create or diminish a GlusterFS trusted storage pool. A set of nodes can be
+    added into an existing trusted storage pool or a new storage pool can be
+    formed. Or, nodes can be removed from an existing trusted storage pool.
+version_added: "2.6"
+author: Sachidananda Urs (@sac)
+options:
+    state:
+       choices: ["present", "absent"]
+       description:
+          - Determines whether the nodes should be attached to the pool or
+            removed from the pool. If the state is present, nodes will be
+            attached to the pool. If state is absent, nodes will be detached
+            from the pool.
+       required: true
+    nodes:
+       description:
+          - List of nodes that have to be probed into the pool.
+       required: true
+    force:
+       choices: ["yes", "no"]
+       default: "no"
+       description:
+          - Applicable only while removing the nodes from the pool. gluster
+            will refuse to detach a node from the pool if any one of the node
+            is down, in such cases force can be used.
+requirements:
+  - PyYAML
```

Comment:
`yaml` doesn't seem used there: is it really a dependency ?

Reply:
That is my mistake, it is not needed. Thanks.